migration/migrator/data/course_tables.sql

```diff
@@ -644,6 +644,18 @@ CREATE TABLE "viewed_responses" (
     CONSTRAINT viewed_responses_pkey PRIMARY KEY ("thread_id", "user_id")
 );
 
+CREATE TABLE socket_connections
+(
+  connection_id SERIAL NOT NULL,
+  user_id       VARCHAR(500),
```

Comment:
`user_id`, as defined elsewhere, should be `character varying(255)`

I am not sure how socket works, so sorry for the dumb question: where is this table used? I cannot find code for inserting records into this table.

Reply:
@zjxiaohan thanks for pointing that out! I'll quickly update that, was hoping to do so in future updates to this PR...also there's no code for that current, I just wanted to see if these changes were failing the build so didn't add a WIP, will do so!